components/util/prefs.rs

```diff
@@ -14,9 +14,9 @@ use std::path::PathBuf;
 use std::sync::{Arc, Mutex};
 
 lazy_static! {
-    static ref PREFS: Arc<Mutex<HashMap<String, Pref>>> = {
+    pub static ref PREFS: Preferences = {
         let prefs = read_prefs().unwrap_or(HashMap::new());
```

Comment:
now you're at it, could you change this `unwrap_or(HashMap::new())` for `unwrap_or_else(HashMap::new)`?

It's a nit though, so feel free to not do it :)